wiki/Disambiguation/fr.md

```diff
@@ -0,0 +1,9 @@
+# Désambiguïsation
+
+La désambiguïsation dans osu! wiki est le processus de résolution des conflits qui surviennent lorsqu'un titre d'article potentiel est ambigu, le plus souvent parce qu'il fait référence à plus d'un sujet couvert par osu! wiki, soit en tant que sujet principal d'un article, soit en tant que sous-thème couvert par un article en plus du sujet principal de l'article.
+## Liste des articles de désambiguïsation
```

Comment:
```suggestion

## Liste des articles de désambiguïsation
```

Reply:
I did like that, not knowing if all the commits would be on a single PR, I still know GitHub badly.